WordPress/Classes/ViewRelated/Media/MediaItemViewController.swift

```diff
@@ -80,7 +80,9 @@ class MediaItemViewController: UITableViewController {
 
                 switch media.mediaType {
                 case .image:
-                    self?.presentImageViewControllerForMedia()
+                    if self?.isMediaLoaded() == true {
```

Comment:
Is there a reason we're only allowing taps into the header if the image is loaded? Previously the image view controller would continue showing the loading state anyway.

Reply:
Yes! Two reasons:
1. It was one of the points we agreed with @iamthomasbishop. Showing loading state in the image full screen is not a problem, but the error state wouldn't play nice. 
2. Other similar sections also wait for the image to load before allowing it to show full screen (i.e. Post settings). So for consistency, we decided to do the same here too.